src/Extractor/LinkExtractorInterface.php

```diff
@@ -0,0 +1,14 @@
+<?php
+/**
+ * @license   http://opensource.org/licenses/BSD-3-Clause BSD-3-Clause
+ * @copyright Copyright (c) 2014 Zend Technologies USA Inc. (http://www.zend.com)
+ */
+
+namespace ZF\Hal\Extractor;
+
+use Zend\Stdlib\Extractor\ExtractionInterface;
```

Comment:
Do we need to extend the `ExtractionInterface`? I ask, because that's mainly designed around the hydrator system, and I'm wondering if having a context-specific interface here might make more sense.